orion_mcp.py

```
@@ -663,6 +753,159 @@ def main():
+    )
+    include_meta = request.query_params.get("include_meta", "0").lower() in ("1", "true", "yes")
+    version = request.query_params.get("version", "4.19")
+
+    if include_meta:
+        try:
+            metrics, meta_map = _load_config_metrics_with_meta(
+                os.path.join(ORION_CONFIGS_PATH, config),
+                version=version,
+            )
+            return JSONResponse({"metrics": metrics, "meta": meta_map})
+        except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)